core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -795,6 +915,12 @@ object ConsumerGroupCommand extends Logging {
                              .withRequiredArg()
                              .describedAs("number-of-offsets")
                              .ofType(classOf[Long])
+    val membersOpt = parser.accepts("members", MembersDoc)
```

Comment:
I did some messing around with joptsimple arguments. There is an `availableIf` function which could be used in this case to declare the dependence on the `--describe` option. Unfortunately, the library is not clever enough to show this dependence in the help message, so all arguments are still organized in a flat list. Since this command is getting complex, a better help message would be a good improvement, but I guess we can take that up in a separate jira.

Reply:
Maybe if we just had a way to show a few common examples, it would be a big help.

Reply:
Sounds good, I'll try to make use of `availableIf` and also provide examples in the help message.